0035-ilp-over-http/0035-ilp-over-http.md

```diff
@@ -40,7 +41,8 @@ Authorization: Bearer zxcljvoizuu09wqqpowipoalksdflksjdgxclvkjl0s909asdf
 
 - **Path** - A connector MAY specify any HTTP path for their peer to send ILP packets to.
 - **Host Header** - The standard [HTTP Host Header](https://tools.ietf.org/html/rfc2616#section-14.23) indicating the domain of the HTTP server the Request is sent to.
-- **Content-Type / Accept Headers** - MUST be set to `application/octet-stream`.
+- **Content-Type / Accept Headers** - MUST be set to `application/octet-stream`.\
+- **ILP-Account-Name** - An optional header used to identify the account for which the request is being made.
```

Comment:
Could you expand on what this header is for? I'm wondering about a couple of points:
- When you say it's optional, does that mean that it's optional from the perspective of this protocol in general or optional from the perspective of every ILP-over-HTTP server? 
- If a server is set up to use this header, how should it behave if that header is not present?
- How do you find out this value? Is that another field that needs to be communicated when peering?
- How do you know whether a peer requires this or not (if it can be required) and do all client implementations need to support sending this field?

Reply:
I think we should either close this PR or else move this header into a section that details (non-normatively) how peers might use extra headers to help with authentication. 

For example, the current Java implementation of ILP-over-HTTP supports a JWT authentication mode where the entire token can trivially be used in a performant manner without _any_ additional headers.

Also, read more about my current opinion here: https://forum.interledger.org/t/connector-auth-model/517/4?u=sappenin

Reply:
Agree. I don't think having this as a normative requirement makes sense.

I do think we should have at least one auth scheme that all implementations MUST support and this should be a well defined standard.

That manifests as some changes to the "Authentication" section to include some normative requirements. I'll write those up.